README.md

```diff
@@ -65,6 +65,31 @@ $ ufmt diff <path> [<path> ...]
 ```
 
 
+`pre-commit` hook
+-----------------
+
+µfmt provides a [`pre-commit`](https://pre-commit.com) hook. To format your diff before 
```

Comment:
think the markdown->rst is choking on backticks here, so maybe just drop them so that the docs site works better: https://ufmt--15.org.readthedocs.build/en/15/